src/tests/address_test.ts

```diff
@@ -269,4 +270,50 @@ describe("Address", () => {
       expect(paginationResponse.nextPage).toEqual("next page");
     });
   });
+
+  describe(".reputation", () => {
```

Comment:
The dot syntax implies a class or static method. The octothorpe syntax represents an instance method!

Reply:
Done